teams/compiler.toml

```diff
@@ -107,6 +107,3 @@ include-subteam-members = true
 
 [[zulip-groups]]
 name = "T-compiler"
-
-[[zulip-groups]]
-name = "T-compiler/meeting"
```

Comment:
We should maybe set `include-team-members` to `false`
https://github.com/rust-lang/team/blob/8d055a5d1693107b41ddd4ea99dc646991e8700c/docs/toml-schema.md?plain=1#L189-L192
and use `extra-people` instead?

Reply:
Setting `include-team-members` to `false`  would be the way it's "supposed to work" in the eyes of the team repo maintainers.

Reply:
I've changed it to this.